### ISSUE TITLE
static_ffmpeg fails to see video file where ffmpeg doesnt

### ISSUE THREAD

**Issue by sh34v3** [zackees/static_ffmpeg#9]
I'm making a subprocess call like so:

```
subprocess.call([
                'static_ffmpeg',                   # call ffmpeg
                '-ss', str(start_time),          # our clip starts here
                '-i', infile_video,         # this is the video to be converted
                '-t', str(end_time - start_time), #how long our clip is
                '-threads', '4',            # use 4 threads for the video conversion
                '-c:v', 'libvpx-vp9',       # c[odec]:v[ideo] - we use libvpx cause we want webm
                '-c:a', 'libvorbis',        # c[odec]:a[udio] - this is the one everyone else was using
                '-b:v',  '400k',            # reccomended video bitrate
                '-b:a', '192k',             # reccomended audio bitrate
                '-deadline', 'good',    # setting for quality vs. speed (best, good, realtime (fastest)); boundry for quality vs. time set the following settings
                '-qmin', '0',               # quality minimum boundry. (lower means better)
                '-qmax', '50',              # quality maximum boundry (higher means worse)
                uniquename               # file to be written out
            ])
```
When I do, I get this error `Some: No such file or directory`. Meanwhile, this works fine:
```
subprocess.call([
                'ffmpeg',                   # call ffmpeg
                '-ss', str(start_time),          # our clip starts here
                '-i', infile_video,         # this is the video to be converted
                '-t', str(end_time - start_time), #how long our clip is
                '-threads', '4',            # use 4 threads for the video conversion
                '-c:v', 'libvpx-vp9',       # c[odec]:v[ideo] - we use libvpx cause we want webm
                '-c:a', 'libvorbis',        # c[odec]:a[udio] - this is the one everyone else was using
                '-b:v',  '400k',            # reccomended video bitrate
                '-b:a', '192k',             # reccomended audio bitrate
                '-deadline', 'good',    # setting for quality vs. speed (best, good, realtime (fastest)); boundry for quality vs. time set the following settings
                '-qmin', '0',               # quality minimum boundry. (lower means better)
                '-qmax', '50',              # quality maximum boundry (higher means worse)
                uniquename               # file to be written out
            ])
```

If it's relevant, the media files are in script's directory, and they end up having filenames with whitespace so their names look like: 'Some Youtube Video.mp4'

**Comment by sh34v3** [comment]
I was able to work around the issue by substituting infile_video for a static complete path to the video that is being converted. That being said, this issue remains.

For reference, here is an example of infile_video: /full/path/to/video/'Youtube Video Title.mp4'

**Comment by zackees** [comment]
Thanks for this bug report.

The quickest work around is to use the api to get the path to the real `ffmpeg` and then just use that for your `subprocess.call`

```
from static_ffmpeg import run
ffmpeg, ffprobe = run.get_or_fetch_platform_executables_else_raise()
subprocess.call([ffmpeg, ...])
```

**Comment by zackees** [comment]
Fixed in v2.2
https://github.com/zackees/static_ffmpeg/commit/b1e150ee126585c93abeee593cc36004adcd6436

Please run
```
python -m pip install static-ffmpeg --upgrade
```

And your issue should be resolved.
